### PR TITLE
bolt documentation

### PR DESCRIPTION
**Pull request by AviAvni**

### CODE REVIEW

File: bolt_support.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+```
+
+### Step 1: Create a `main.py` File
+
+Create a main.py file with the following content:
+
+```python
+from neo4j import GraphDatabase
+
+driver = GraphDatabase.driver("bolt://localhost:7687", auth=("falkordb", ""))
```
Comment (swilly22) [inline bolt_support.md:26]: When running either a docker image of `FalkorDB` or when running in the cloud, what's the default credentials?
is it Username:'falkordb' and password:'' ?
If that's the case I think we need to change it, if not this documentation should indicate that the user creds' need to be specified, e.g. `<username>, <password>`

File: bolt_support.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+records, _, _ = driver.execute_query(
+    "UNWIND range(1, $n) AS i RETURN i",
+    n=10, database_="mygraph",
+)
```
Comment (swilly22) [inline bolt_support.md:31]: except for `records` are the addition two discarded agreement are of any value ? (what are they) ?
I find it a bit annoying that the user need to specify the `database` for each query, moreover why is the parameter called `database_` and not `database` ?

File: bolt_support.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+### Step 2: Run the script
+
+Execute the script by running the following command in your terminal:
+
+bash
+```bash
+python main.py
+```
+
+Adjust the authentication parameters and database name according to your FalkorDB setup. This script demonstrates a simple query that returns the numbers from 1 to 10. Customize the query as needed for your specific use case.
```
Comment (swilly22) [inline bolt_support.md:46]: This is the explanation I was missing earlier.
I think this sentence should be a bit rephrased and placed before the code snippet.

**Review by swilly22 — CHANGES_REQUESTED** [review]

File: bolt_support.md (hunk 2 above)
Comment (AviAvni) [reply bolt_support.md:31]: https://pypi.org/project/neo4j/ take a look at the first example

Comment (AviAvni) [reply bolt_support.md:31]: they can just read the doc for this driver

**Review by swilly22 — APPROVED** [review]

**Pull request merged**